### PR TITLE
6927: updated button for right sidebar

### PR DESCRIPTION
**Pull request by nikola-gladovic**
Signed-off-by: Nikola <nikola.gladovic@nextcloud.com>

#6927

### CODE REVIEW

File: src/components/RightSidebar/RightSidebar.vue [hunk 1]
```diff
@@ -69,9 +69,9 @@
 				:name="conversation.displayName" />
 			<div v-if="!getUserId" id="app-settings">
 				<div id="app-settings-header">
-					<button class="settings-button" @click="showSettings">
+					<Button type="tertiary" class="settings-button" @click="showSettings">
```
Comment (marcoambrosini) [inline src/components/RightSidebar/RightSidebar.vue:72]: class to be removed, at least for the styling of the element

Screenshot 2022-03-07 at 18 09 54

**Review by marcoambrosini — CHANGES_REQUESTED** [review]

**Review by marcoambrosini — CHANGES_REQUESTED** [review]
Perfect, now you need to pass in a material design icon in the icon slot to replace the old one

Screenshot 2022-03-09 at 08 59 22

File: src/components/RightSidebar/RightSidebar.vue (hunk 1 above)
Comment (nikola-gladovic) [reply src/components/RightSidebar/RightSidebar.vue]: Should I remove the class from new Buttons? :)

**Review by nickvergessen — APPROVED** [review]

**Comment by nickvergessen** [comment]
Good now @marcoambrosini ?

**Review by marcoambrosini — APPROVED** [review]

**Pull request merged**